Source/Extensions/Blazorise.DataGrid/_DataGridRow.razor

```diff
@@ -22,6 +22,20 @@
               Class="@GetClass(styling, selectedStyling)"
               Style="@GetStyle(styling, selectedStyling)"
               HoverCursor="@GetHoverCursor()">
+
+        @if ( ParentDataGrid.IsGroupable() )
+        {
+            var count = ParentDataGrid.GroupableColumnsCount();
+            if ( count > 1 )
+            {
+                for ( int i = 0; i < count; i++ )
+                {
+                    <TableRowCell Style="width:25px;" />

```

Comment:
What is the need for a cell with `Style="width:25px;"`?

Reply:
I was doing that so the width was predictable and then we could use width times the number of nesting to calculate the indenting.
But I guess I'm a bit lost with the indenting part. If you can take a look.

Reply:
I can look at it once you settle on the almost-finished solution.

Reply:
Right maybe I can think of something in the meantime. Thanks.

Reply:
Try with Margin or Padding utility.

Reply:
It didn't work because it applies to the TR which has no effect.

Reply:
I mean, at least from what I tried